src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java

```diff
@@ -326,20 +314,20 @@ private long getJwtExpiration(SamlResponse samlResponse) throws Exception {
         DateTime sessionNotOnOrAfter = samlResponse.getSessionNotOnOrAfter();
 
         if (this.expiryBaseValue == ExpiryBaseValue.NOW) {
-            return System.currentTimeMillis() / 1000 + this.expiryOffset;
+            return System.currentTimeMillis() + this.expiryOffset * 1000;
```

Comment:
Umm, this looks like it could be a bug - we should have test that confirms this behavior.

Reply:
No, this isn't a bug, since the new library is taking the time as a `Date` type, which is in milliseconds. When we set up the expiry, it takes something like this:
```
/**
 * Sets the expiration time ({@code exp}) claim.
 *
 * @param exp The expiration time, {@code null} if not
 *            specified.
 *
 * @return This builder.
 */

public Builder expirationTime(final Date exp) {

  claims.put(JWTClaimNames.EXPIRATION_TIME, exp);
  return this;
}
```
So all the tests with Authentication should cover this change, since we do need the correct expiry to authenticate.